crates/shielded_token/src/vp.rs

```diff
@@ -869,7 +869,7 @@ fn apply_balance_component(
     })
 }
 
-// Verify that the pre balance + the Sapling value balance = the post balance
+// Verify that the pre balance - the Sapling value balance = the post balance
```

Comment:
Was logic actually changed here, or was this comment just wrong before?

Reply:
Wrong comment but the logic is unchanged